README.md

```diff
@@ -94,67 +27,92 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+module "ibm_cbr" "zone" {
+  # replace master with version
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cbr//cbr-zone-module?ref=master"
```

Comment:
in public primary branch is `main`, so replace `master` with `main` here and in the comment above it